### ISSUE TITLE
Is this still being maintained?

### ISSUE THREAD

**Issue by jmbowman** [brutasse/django-ratelimit-backend#53]
It looks like there haven't been any commits in 20 months, and there are a few reasonable-looking PRs that haven't received any reply.  @brutasse , are you still maintaining this repository?  If not, are you open to handing over to a new maintainer and/or helping facilitate a clean transition to a new fork?

Asking because I'm currently using a fork that we had hoped would be temporary, but are now looking at needing to update it for new Django versions, CI improvements, etc. (and our last submitted PR is among the ones awaiting a reply).

**Comment by brutasse** [comment]
@jmbowman thanks, indeed I'm happy to transfer ownership to a maintained fork. Is the EDX fork the one you're referring to?

**Comment by jmbowman** [comment]
Yes; I just kicked off an internal conversation about potentially adopting this, hoping to have a consensus on it by next week.  That'll also give some time for anybody currently using another fork to chime in with any suggestions or questions.

**Comment by brutasse** [comment]
Thanks!

**Comment by jmbowman** [comment]
We learned a lot from that internal conversation!  I'll link to our findings and subsequent code changes, as they may be useful to other users of this package:

* https://github.com/edx/edx-platform/blob/master/docs/decisions/0009_simplify_ratelimiting.rst
* https://github.com/edx/edx-platform/pull/26487

Basically, we found out that our login rate limiting in that service was a confusing pile of overlapping implementations.  At least for our use case, [django-ratelimit](https://pypi.org/project/django-ratelimit/) covers what we need (and was already in use for other URLs), so I'm afraid we'll be dropping our fork of this package.  The notes above may help serve as a migration path for others.

**Comment by brutasse** [comment]
Thanks @jmbowman, interesting read :+1:
